packages/cdktf-cli/bin/cmds/deploy.ts

```diff
@@ -2,11 +2,12 @@ import * as yargs from "yargs";
 import { config as cfg } from "@cdktf/provider-generator";
 import { requireHandlers } from "./helper/utilities";
 import { Errors } from "../../lib/errors";
+import { BaseCommand } from "./helper/base-command";
 
 const config = cfg.readConfigSync();
 
-class Command implements yargs.CommandModule {
-  public readonly command = "deploy [OPTIONS] <stacks..>";
+class Command extends BaseCommand {
+  public readonly command = "deploy [stacks...]";
```

Comment:
The brackets here puzzle me, do you know their respective meaning?

Reply:
Yes.. Figuring that out took the most amount of time 😅
Yargs does not support `required: false` as a setting on `.positional()` but it does support specifying them using `[]` instead of `<>` which switches between them being optional or required 😁
Also: The TS types for `.positional()` are wrong, they mention `required` but yargs does not support that there..